src/main/java/competition/subsystems/armlifting/commands/FrontGrabbingBallsCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.armlifting.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.armlifting.FrontCollectingSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class FrontGrabbingBallsCommand extends BaseCommand {
+
+    final FrontCollectingSubsystem frontCollectingSubsystem;
+    final OperatorInterface oi;
+
+    @Inject
+    public FrontGrabbingBallsCommand(OperatorInterface oi, FrontCollectingSubsystem frontCollectingSubsystem) {
+        this.oi = oi;
+        this.frontCollectingSubsystem = frontCollectingSubsystem;
+        this.requires(this.frontCollectingSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+    }
+
+    @Override
+    public void execute() {
+        frontCollectingSubsystem.intake();
+        frontCollectingSubsystem.stop();
```

Comment:
This will intake the motor for ~1/50th of a second, then stop it. When intaking like this, you probably want to do it continuously. (remove the `stop()` call.)

Reply:
done